### PR TITLE
Windows build

### PR DESCRIPTION
**Pull request by MrYadro**
Please, add AppVeyor project to enable ci.

### CODE REVIEW

**Comment by MrYadro** [comment]
Extras https://github.com/MrYadro/xash3d/releases

File: engine/client/cl_main.c [hunk 1]
```diff
@@ -923,7 +923,7 @@ void CL_InternetServers_f( void )
 		MsgDev( D_INFO, "Can't resolve adr: %s\n", MASTERSERVER_ADR );
 
 	Q_snprintf( fullquery, sizeof( fullquery ),
-		"\x31\xFF0.0.0.0:0\0"
+		"\x31\xFF""0.0.0.0:0\0",
```
Comment (a1batross) [inline engine/client/cl_main.c]: Why there is two "? An extra comma here, it doesn't need.

File: engine/engine.vcxproj [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v120</PlatformToolset>
+    <PlatformToolset>v120_xp</PlatformToolset>
     <CharacterSet>MultiByte</CharacterSet>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v120_xp</PlatformToolset>
+    <PlatformToolset>v120</PlatformToolset>
```
Comment (a1batross) [inline engine/engine.vcxproj]: Maybe build Release with xp support too?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-# Xash3D Engine [![Build Status](https://travis-ci.org/SDLash3D/xash3d.svg)](https://travis-ci.org/SDLash3D/xash3d)
-
+# Xash3D Engine [![Build Status](https://travis-ci.org/SDLash3D/xash3d.svg)](https://travis-ci.org/SDLash3D/xash3d) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/github/SDLash3D/xash3d?branch=master&svg=true)](https://ci.appveyor.com/project/SDLash3D/xash3d/) [![Join the chat at https://gitter.im/MrYadro/xash3d](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/SDLash3D/xash3d?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (a1batross) [inline README.md]: gitter.im? >_> We have a Jabber conference for it. :)

**Pull request merged**